crates/holochain_types/src/app.rs

```diff
@@ -56,7 +56,7 @@ pub enum CoordinatorSource {
 #[derive(Debug, serde::Serialize, serde::Deserialize)]
 pub struct RegisterDnaPayload {
     /// Modifier overrides
-    pub modifiers: DnaModifiersOpt<YamlProperties>,
+    pub modifiers: Option<DnaModifiersOpt<YamlProperties>>,
```

Comment:
I know you just changed a bunch of occurrences, but I wonder if it would be better to just use serde default? We don't care to distinguish between the `None` case and the `Default` case.

```suggestion
    #[serde(default)]
    pub modifiers: DnaModifiersOpt<YamlProperties>,
```

Reply:
Absolutely. I didn't know it was that easy